docs/README.md

```diff
@@ -122,6 +122,10 @@ OTEL_DOTNET_AUTO_INTEGRATIONS_FILE=%InstallationLocation%/integrations.json
 
 See [config.md](config.md).
 
+### Manual instrumentation
```

Comment:
Should we use a different heading here, we might conflict with manual instrumentation defined in OpenTelemetry specification - https://opentelemetry.io/docs/concepts/instrumenting/#manual-instrumentation

Reply:
@rajkumar-rangaraj, do you  have any suggestion for the naming?
I agree that we need a subset of steps defined on the linked site, but I would like to avoid calling it semi-manual/semi-auto instrumentation.

Reply:
I think that manual instrumentation is appropriate here. Our documentation about manual instrumentation aligns with the info in https://opentelemetry.io/docs/concepts/instrumenting/#manual-instrumentation. I think that the bigger difference is that for .NET, the OpenTelemetry API is implemented in DiagnosticsSource (which our users may not be aware of). The main difference that I see is that they don't have to follow the information about the SDK.